test/transports/01-file-maxsize.test.js

```diff
@@ -60,10 +60,10 @@ describe('File (maxsize)', function () {
         }
 
         const text = fs.readFileSync(file, 'utf8');
-        // Fails to assert the 5th file because it's empty.
         assume(text[0]).equals(fillWith[i]);
-        // Fails on each file because the eol also adds to the size.
-        assume(stats.size).equals(4096);
+        // Either 4096 on Unix or 4100 on Windows
+        // because of the eol.
+        assume(stats.size).either([4096, 4100]);
```

Comment:
Can we get the value based on the platform then assert it? If we get `4096` on Windows it's a bug and if we get `4100` on *nix it's a bug. e.g.:

``` js
// Either 4096 on Unix or 4100 on Windows because of the eol
const expected = process.platform === 'win32' ? 4100 : 4096;
assume(stats.size).equals(expected);
```